### ISSUE TITLE
Drag 'n drop modules to load them

### ISSUE THREAD

**Issue by SuperDisk** [stoneface86/trackerboy#20]
Would be pretty handy to be able to drag `tbm` files into the window to load them. I find myself doing this by habit.

**Comment by stoneface86** [comment]
Agree! This is one of those things I've been meaning to add but keep forgetting to.
